.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# OpenAPI CLI Contributing Guide
```

Comment:
please add a link to this file to the main README.md

Reply:
Added.